.buildkite/x-pack/pipeline.xpack.filebeat.yml

```diff
@@ -462,3 +462,24 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/filebeat: Packaging Linux arm64"
+
+      - label: ":linux: x-pack/filebeat: Packaging Linux FIPS"
+        key: "packaging-linux-fips"
+        env:
+          PLATFORMS: "+all linux/amd64 linux/arm64"
```

Comment:
ditto.

Reply:
Added PLATFORMS_*_FIPS in  394149f7e2f796040aef963390df1d7a437c5f21